Library/Formula/percona-toolkit.rb

```diff
@@ -1,3 +1,10 @@
+class AnyMySQLRequirement < Requirement
+  fatal true
+  default_formula "mysql"
+
+  satisfy { which("mysql_config") }
+end
```

Comment:
Yeah. I was thinking of this sort of thing but without setting a `default_formula`. If it won't build in that configuration we'll need to go down the bottle disable route I guess, per Mike's earlier comment.